services/zendesk.rb

```diff
@@ -1,18 +1,17 @@
-require 'zendesk2'
+require 'zendesk_api'
 require 'date'
 
 class Zendesk
   def initialize
-    options = {
-      :username => SupportApp.zendesk_username,
-      :token => SupportApp.zendesk_token,
-      :url => SupportApp.zendesk_url
-    }
-    @client = Zendesk2::Client.new(options)
+    @client = ZendeskAPI::Client.new do |config|
+      config.username = SupportApp.zendesk_username
+      config.token = SupportApp.zendesk_token
+      config.url = SupportApp.zendesk_url
+    end
   end
 
   def incidents_for_the_past_week
-    incidents_group = 'Incident response and tuning'
-    @client.tickets.search(query: "created>7days group:\"#{incidents_group}\"").count
+    incidents_group = 'Incident Response'
```

Comment:
We don't need to interpolate this variable as its not dynamic. This query string can just be a constant.

```ruby
  SOME_COOL_QUERY_NAME = \
    'type:ticket ticket_type:incident created>"7 days ago" group:\"Incident Response"'.freeze

def inicidents_for_the_past_week
  @client.search(query: SOME_COOL_QUERY_NAME).count
end
```